Packages/Tags/Runtime/AtomTags.cs

```diff
@@ -172,6 +175,9 @@ public void AddTag(StringConstant tag)
             // Update static accessors:
             if (!TaggedGameObjects.ContainsKey(tag.Value)) TaggedGameObjects.Add(tag.Value, new List<GameObject>());
             TaggedGameObjects[tag.Value].Add(this.gameObject);
+
+            // Start TagAdded event
+            TagAdded?.Invoke(tag.Value);
```

Comment:
When using StringEvent, this will be `Raise` instead of `Invoke`.
Also StringEvent is a UnityEngine.Object type, so it must not be compared with `?.` but instead using `if(... == null)` because the UnityEngine.Object `==` operator is overloaden and this is not taken into account when using `?.` or `is null`